### ISSUE TITLE
Newbie (Game refusing to TEST)

### ISSUE THREAD

**Issue by Totally2d** [bladecoder/bladecoder-adventure-engine#34]
Hi,

First and foremost, your engine is AMAZING! I can not say that enough, I'm just impressed with the features I can see (depth of field and stuff). Plus, I played the Android games from the App Store.

 I have a challenge with my first attempt at using the engine. I've used Construct2 and Adventure Game Studio but Bladecoder is new to me.

My challenge is, I'm learning by doing. So I've started a game. Unfortunately this game wont do a "Test" so that I can see how things are running. It's giving me the errors I've attached in the images below. Are you able to assist me in layman terms, how to fix this? The solutions I'm finding on Google, either don't apply to me or they have a high skill level of assumed knowledge. 

I can see that this seems to be JAVA related, but any help you can give me would be greatly useful.

 I feel so stuck!

![1](https://user-images.githubusercontent.com/36259753/35974985-d7399f20-0ce3-11e8-8c6d-dc97d73ec5f2.jpg)
![2](https://user-images.githubusercontent.com/36259753/35974986-d76ea706-0ce3-11e8-8e5c-6566ed7be2f1.jpg)

**Comment by bladecoder** [comment]
Hi, @Totally2d,

I need more info to understand the problem:

- The screenshots are almost the same, can you put the missing part of the top, please?
- Which version of the engine are you using? Can you attach the gradle.properties of you project?

Cheers!

**Comment by Totally2d** [comment]
Hi,

Thank you for the prompt reply. I've never worked with Gradle before, so let me know if the info I got helps. 

I've attached images of the entire message. 

Plus I checked the Gradle file I downloaded and installed today its "gradle-core-2.3.0". I also found a file named (gradle.properties) which has this info:

#Thu Feb 08 14:39:40 CAT 2018
roboVMVersion=2.3.3
spinePlugin=false
gdxVersion=1.9.8
org.gradle.daemon=true
bladeEngineVersion=2.0.0
org.gradle.jvmargs=-Xms128m -Xmx1536m
bladeInkVersion=0.4.4
roboVMGradlePluginVersion=2.3.3
version=1
appName=Testingblade
org.gradle.configureondemand=true
androidGradlePluginVersion=2.3.0

![3](https://user-images.githubusercontent.com/36259753/35987749-43604d86-0d05-11e8-957a-2dff8744f52a.jpg)

![4](https://user-images.githubusercontent.com/36259753/35987767-52a09044-0d05-11e8-81e0-64a23e59e998.jpg)

![5](https://user-images.githubusercontent.com/36259753/35987798-647e9a86-0d05-11e8-8d81-f948d8a8ec80.jpg)

![6](https://user-images.githubusercontent.com/36259753/35987814-7164e688-0d05-11e8-8dc6-6f56e8e16175.jpg)

**Comment by bladecoder** [comment]
Ok, the problem is with your java install. You have installed the JRE but you NEED the JDK.

You have to uninstall your Java and download and install the JDK here: http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html

**Comment by Totally2d** [comment]
Thank you so much!!!! Let me do that and tell you what happens.

**Comment by Totally2d** [comment]
My internet connection is a bit slow, so I'll keep the download running as long as it but as soon as it's finished I'll let you know the outcome. 

Thank you again for such an amazing engine. 

I still have a lot to learn but I'm looking for every Bladecoder tutorial video I can find.
My first games with your engine are oing to be short simple, so that I can see how things work. 

I'll be in touch when my download & install ends.

Tafadzwa

**Comment by Totally2d** [comment]
Thank you!!!

The problem has been solved 😃

I can now resume my learning!

**Comment by bladecoder** [comment]
That's great. I close the issue.
